esp/services/ws_sql/SQL2ECL/SQLColumn.hpp

```diff
@@ -166,7 +168,21 @@ class HPCCColumnMetaData : public CInterface, public IInterface
 
     void setColumnType(const char* columnType)
     {
-        this->columnType.set(columnType);
+        if (strncmp(columnType, "table of", 8)==0)
+        {
+            StringBuffer result;
+			result.append("DATASET({");
```

Comment:
minor: weird indentation. - including the else.  Tabs being used instead of spaces.